src/AppKit/NSAccessibility.cs

```diff
@@ -80,6 +89,93 @@ public static CGPoint GetPointInView (NSView parentView, CGPoint point)
 		{
 			return NSAccessibilityPointInView (parentView, point);
 		}
+
+		[Mac (10,7)]
+		[DllImport (Constants.AppKitLibrary)]
+		static extern void NSAccessibilityPostNotificationWithUserInfo (IntPtr element, IntPtr notification, IntPtr userInfo);
+
+		public static void PostNotification (NSObject element, NSString notification, NSDictionary userInfo)
+		{
+			NSAccessibilityPostNotificationWithUserInfo (element.Handle, notification.Handle, userInfo.Handle);
+		}
+
+		[DllImport (Constants.AppKitLibrary)]
+		static extern void NSAccessibilityPostNotification (IntPtr element, IntPtr notification);
+
+		public static void PostNotification (NSObject element, NSString notification)
+		{
+			NSAccessibilityPostNotification (element.Handle, notification.Handle);
+		}
```

Comment:
I believe you need to check against null ptr here too. element and notification can be null and you will get a NRE.